autoload/fzf/OmniSharp.vim

```diff
@@ -24,10 +24,11 @@ function! fzf#OmniSharp#FindSymbols(quickfixes) abort
   for quickfix in s:quickfixes
     call add(symbols, s:format_line(quickfix))
   endfor
-  call fzf#run({
+  let fzf_options = len(g:OmniSharp_fzf_options) ? copy(g:OmniSharp_fzf_options) : { 'down': '40%' }
```

Comment:
I think this might be simpler if we use `get()`, as it doesn't require `g:OmniSharp_fzf_options` to already have been initialised, and it takes a default as an argument:

```vim
let fzf_options = copy(get(g:, 'OmniSharp_fzf_options', { 'down': '40%' }))

Reply:
Ah yeah that seems cleaner, thanks!